admin/templates/Includes/LeftAndMain_Menu.ss

```diff
@@ -1,36 +1,10 @@
 <div class="cms-menu cms-panel cms-panel-layout west" id="cms-menu" data-layout-type="border">
 	<div class="cms-logo-header north">
```

Comment:
Ah okay I see so its about overriding the LeftAndMain_Menu.ss rather than the custom components which makes sense. Sorry I didn't think about flipping it that way around :smile: